unified_planning/model/fluent.py

```diff
@@ -35,6 +35,16 @@ def __init__(self, name: str, typename: 'unified_planning.model.types.Type' = No
             self._typename = typename
         self._signature = signature
 
+    ##ADDED###
```

Comment:
This should NOT be needed. A Fluent is a persistent (aka immutable) data structure, once created you cannot change it. So deep-copying it is never needed